mage_ai/data_preparation/models/variable.py

```diff
@@ -273,6 +302,20 @@ async def write_data_async(self, data: Any) -> None:
         else:
             await self.__write_json_async(data)
 
+        self.write_metadata()
+
+    def write_metadata(self) -> None:
```

Comment:
we use another [`metadata.json`](https://github.com/mage-ai/mage-ai/blob/master/mage_ai/data_preparation/models/variable.py#L607) to store dataframe stats. can we clarify the difference here?

Reply:
ok i wasn't sure if that was still being used. i can change the name of the new file